app/src/main/kotlin/eu/quiqua/gloomhaven/statstracker/features/stats/viewModel/StatsViewModel.kt

```diff
@@ -17,7 +20,42 @@ class StatsViewModel @Inject constructor(application: android.app.Application) :
     }
 
     @Inject
-    lateinit var model: String
+    lateinit var stats: Stats
 
-    fun getStats(): String = model
+    val mutableHp: MutableLiveData<Int> = MutableLiveData()
```

Comment:
can't you have livedata with your model directly?

Reply:
maybe :/ I don't know yet how this works or how to apply this in the correct way. Difficult to find good examples online :)